### DISCUSSION TITLE
RedHat (Fedora / CentOS / Alma / Rocky) Version of the Server Apps?

### DISCUSSION THREAD

**Discussion by alatark** [rustdesk/rustdesk-server#415] [Q&A]
For the client, I see there are .rpm files for both Suse and RedHat flavours of Linux.

However, I cannot locate .rpms for the server apps (hbbr / hbbs).

Can I simply download the exes in https://github.com/rustdesk/rustdesk-server/releases/download/1.1.10-3/rustdesk-server-linux-amd64.zip and set them up as services?

Is there any intention to create rpms? If not, are there known obstacles to me building them from source for these OSes?

**Comment by paspo** [comment]
if you can create a PR, that'll be great.
Ideally you have to change [the build workflow](https://github.com/rustdesk/rustdesk-server/blob/master/.github/workflows/build.yaml)
